ide/idea/vcs.xml

```diff
@@ -10,6 +10,14 @@
           <option name="issueRegexp" value="CASSANDRA-(\d+)" />
           <option name="linkRegexp" value="https://issues.apache.org/jira/browse/CASSANDRA-$1" />
         </IssueNavigationLink>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="CNDB-(\d+)"/>
```

Comment:
could you try if `(?i)CNDB-(\d+)` enables case insensitive matches? I think it would be better to have case insensitive regexp